e2e/Tests/iothub/device/FaultInjectionPoolAmqpTests.IncomingMessageFaultInjectionPoolAmqpTests.cs

```diff
@@ -188,25 +186,26 @@ public async Task Message_DeviceSak_GracefulShutdownReceiveUsingCallbackRecovery
 
             async Task InitOperationAsync(IotHubDeviceClient deviceClient, TestDevice testDevice, TestDeviceCallbackHandler testDeviceCallbackHandler)
             {
-                using var cts = new CancellationTokenSource(TimeSpan.FromSeconds(20));
-                await deviceClient.OpenAsync(cts.Token).ConfigureAwait(false);
-
-                await testDeviceCallbackHandler.SetMessageReceiveCallbackHandlerAndCompleteMessageAsync<string>().ConfigureAwait(false);
+                using var subscribeCallbackCts = new CancellationTokenSource(s_defaultOperationTimeout);
+                await testDeviceCallbackHandler.SetIncomingMessageCallbackHandlerAndCompleteMessageAsync<string>(subscribeCallbackCts.Token).ConfigureAwait(false);
             }
 
             async Task TestOperationAsync(IotHubDeviceClient deviceClient, TestDevice testDevice, TestDeviceCallbackHandler testDeviceCallbackHandler)
             {
-                OutgoingMessage msg = OutgoingMessageHelper.ComposeOutgoingTestMessage(out string payload, out string p1Value);
+                OutgoingMessage msg = OutgoingMessageHelper.ComposeTestMessage(out string payload, out string p1Value);
                 testDeviceCallbackHandler.ExpectedOutgoingMessage = msg;
-                await serviceClient.Messages.SendAsync(testDevice.Id, msg).ConfigureAwait(false);
 
-                using var cts = new CancellationTokenSource(TimeSpan.FromSeconds(20));
-                await testDeviceCallbackHandler.WaitForReceiveMessageCallbackAsync(cts.Token).ConfigureAwait(false);
+                using var sendMessageCts = new CancellationTokenSource(s_defaultOperationTimeout);
```

Comment:
Why have small timeouts for each call, rather than one total test timeout in a CTS that each call uses?

Reply:
The fault injection tests require different timeouts for different calls, but I could consolidate the timeout for non-fault injection tests.